app/models/docker_container_wizard_states/environment.rb

```diff
@@ -2,6 +2,8 @@ module DockerContainerWizardStates
   class Environment < ActiveRecord::Base
     self.table_name_prefix = 'docker_container_wizard_states_'
     belongs_to :wizard_state, :class_name => DockerContainerWizardState
+
+    attr_accessible :tty, :docker_container_wizard_state_id, :attach_stdin, :attach_stdout, :attach_stderr
```

Comment:
Looks like this line is too long:

http://ci.theforeman.org/job/test_plugin_pull_request/1846/database=sqlite3,label=fast,ruby=2.1/console